.gitignore

```diff
@@ -1,11 +1,17 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
 # directories
-/demo/dist
 /es
 /lib
 /node_modules
 /umd
+/*/**
+!/demo/**
+/demo/dist
```

Comment:
New `demo` directory has it's own package.json and .gitignore and whatnot since it's a CRApp project– any references to `demo/dist` shouldn't exist anymore (they're `build` in crapp, and handled in that demo directory's gitignore)

Reply:
Fixed